.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - lts/*
```

Comment:
It wouldn't hurt to test against both latest stable and lts. While we use lts in our build servers we direct people to install the latest node as part of their machine setup process.